tests/h/search/query_test.py

```diff
@@ -11,12 +11,11 @@ class TestTopLevelAnnotationsFilter(object):
 
     def test_it_filters_out_replies_but_leaves_annotations_in(self, Annotation, search):
         annotation = Annotation()
-        reply = Annotation(references=[annotation.id])
+        Annotation(references=[annotation.id])
 
         result = search.run({})
 
-        assert annotation.id in result.annotation_ids
```

Comment:
I got rid of the explicit check to make sure the reply was not in the results as using a complete list to check against does the same thing in one line.